dashboard/app/models/levels/standalone_video.rb

```diff
@@ -60,4 +60,22 @@ def enable_scrolling?
   def self.create_from_level_builder(params, level_params)
     create!(level_params.merge(user: params[:user], game: Game.standalone_video, level_num: 'custom'))
   end
+
+  def localized_long_instructions
+    return nil unless long_instructions
+    return long_instructions if I18n.en?
+    return I18n.t(name,
+      scope: [:data, "long_instructions"],
+      default: long_instructions,
```

Comment:
oh, please also add `smart: true` to the `I18n.t` call here and below. It's necessary when dealing with keys that can have periods (as is the case for level names)

Reply:
Will do. Out of curiosity, why isn't `smart: true` the default?

Reply:
Just because I didn't want to risk unintentional consequences when we originally added it